README.md

```diff
@@ -1,68 +1,85 @@
 [![Build Status](https://travis-ci.org/Adyen/adyen-python-api-library.svg?branch=master)](https://travis-ci.org/Adyen/adyen-python-api-library)
 [![Coverage Status](https://coveralls.io/repos/github/Adyen/adyen-python-api-library/badge.svg?branch=master)](https://coveralls.io/github/Adyen/adyen-python-api-library?branch=master)
 
-# Adyen APIs Library for Python
```

Comment:
Why is this being removed?

Reply:
Not sure but these changes are stale now. Will not include them when I reopen.